packages/flutter_migrate/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
+
+- Initial version.
```

Comment:
Nit: Prefer `*` for new packages, per style guide.

Reply:
hahaha ok I was confused... and brainlessly applied what you said as I thought you knew something I didnt *facepalm*